sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -659,6 +659,14 @@ case class ShowTables(
     AttributeReference("tableName", StringType, nullable = false)())
 }
 
+/**
+ * The logical plan of the USE/USE CATALOG command that works for v2 catalogs.
+ */
+case class UseCatalogAndNamespace(
```

Comment:
let's make the name clearer: `SetCurrentCatalogAndNamespace`.

Reply:
Changed.